EnglishTranslation-master/QUEST_LV_0200.tsv

```diff
@@ -7380,13 +7380,13 @@ QUEST_LV_0200_20150714_007383	The information should sometimes be leaked, hid or
 QUEST_LV_0200_20150714_007384	People say history is the record of the winners. I think so too.{nl}Have you seen the tombstones that are still remaining on the ruins near Fedimian?{nl}
 QUEST_LV_0200_20150714_007385	If what the soldiers are spreading is true, then there are many doubts on the records that are remaining here.{nl}And the behaviors of the soldiers that are trying to destroy them are also strange.{nl}{nl}
 QUEST_LV_0200_20150714_007386	Throw this against the monsters.{nl}If you are lucky enough, it will explode with a bang!, but if that's not it, you will make them angry.
-QUEST_LV_0200_20150714_007387	Please collect the dark crystals from the monsters at Vienti Fortress.{nl}I will try to make an elixir that would pull out the controlled spirit.
+QUEST_LV_0200_20150714_007387	Please collect the Dark Crystals from the monsters at Vienti Fortress.{nl}I will try to make an elixir that would pull out the controlled spirit.
 QUEST_LV_0200_20150714_007388	Thanks. First, would you use this melting solution to the monsters that are struggling due to the Stone Frosts?{nl}I mean Nebekia Fortress. If it's useful, you would be able to use it on the humans as well.
 QUEST_LV_0200_20150714_007389	I personally think the curse of the petrification is a chemical reaction.{nl}Which means you can't restore the things that are once petrified before.{nl}
 QUEST_LV_0200_20150714_007390	Even if, Bokor over there tries his best, he won't be able to restore it.{nl}
 QUEST_LV_0200_20150714_007391	But, if the petrification is in progress,{nl}Then I think we may able to save the rest even if we lost the parts that have already become petrified.{nl}
-QUEST_LV_0200_20150714_007392	Almost all curses disappear when the casters die.{nl}I think the curse of the petrification is the same as the case in the way of the pilgrim.{nl}
-QUEST_LV_0200_20150714_007393	But the curse descended for 600 years.{nl}This means the caster of the curse also has lived for 600 years.{nl}
+QUEST_LV_0200_20150714_007392	Almost all curses will wear off when the one who casted it dies.{nl}That's why I think the curse of the petrification is similar to Pilgrim's Way.{nl}
```

Comment:
That's why I think the curse of the petrification is similar to (the curse) at Pilgrim's Way.